File: food_ordering_app/lib/models/review.dart
class Review {
  final int id;
  final int foodId;
  final int userId;
  final String comment;
  final double rating;

  Review({required this.id, required this.foodId, required this.userId, required this.comment, required this.rating});

  factory Review.fromJson(Map<String, dynamic> json) {
    return Review(
      id: json['id'],
      foodId: json['food_id'],
      userId: json['user_id'],
      comment: json['comment'],
      rating: json['rating'].toDouble(),
    );
  }

  get user => null;
}

File: food_odering-backend/app/Models/Review.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Review extends Model
{
    use HasFactory;

    protected $fillable = ['food_id', 'user_id', 'rating', 'comment'];

    // Quan hệ với Food
    public function food()
    {
        return $this->belongsTo(Foods::class);
    }

    // Quan hệ với User
    public function user()
    {
        return $this->belongsTo(User::class);
    }
}
